_sarah.l.fong.github.io_
# Sarah Fong

Ph.D. candidate in human genetics at Vanderbilt University. In the Capra lab (Tony is now at UCSF), I study the evolution of human gene regulation, functional genomics, and how it affects human phenotypes. 

I am interested in applying principles of genetics, functional genomics, and computational approaches to develop novel therapies for cancer, hematopoietic, and immune-mediated diseases. 

# Experience

## The evolution of functional human gene regulatory enhancers  
### Ph.D. dissertation at Vanderbilt University in Nashville, Tennessee 
I study the evolution of human gene regulatory elements, its relationship to functional genomics, and effects on human phenotypes. Under the guidance of Dr. John A. (Tony) Capra (now at UCSF), I research the evolutionary histories of human enhancer sequences using computational approaches. From this work, we identified that transcribed human enhancer sequences have evolved by two evolutionary trajectories. In one, functional enhancer sequences are gained at a discrete evolutionary period and conserved in the human genome. In another, enhancer sequences are gained at one discrete evolutionary period, but expand to include other younger sequences over time. These different evolutionary histories are associated with differences in gene regulatory function, susceptibilities to genetic variants that affect biochemical activity, human variation, and transposable element insertions.

**Relevant skills** 
- Functional genomics: Analyses of ChIP-seq, ATAC-seq, and massively parallel reporter assays, gene regulatory annotation strategies.

- Evolutionary genetics: multiple sequences alignments, evolutionary sequence conservation, human acceleration.

- Human genetic varitation: human population genetics, eQTL and GWAS analyses, electronic health records.


- Computational methods: machine learning strategies, high-performance compute clusters, jupyter notebooks, BEDTools, python, R, bash, and associated statistical methods. 

**Publications**
- [Modeling the Evolutionary Architectures of Transcribed Human Enhancer Sequences Reveals Distinct Origins, Functions, and Associations with Human Trait Variation](https://pubmed.ncbi.nlm.nih.gov/33973014/)
- [Function and constraint in enhancer sequences with multiple evolutionary origins](https://www.biorxiv.org/content/10.1101/2022.01.05.475150v1)

**github resources** 
- [Scripts](https://github.com/slifong08/tools) I use to study the genome and evolution of human enhancer sequences.

## Developing antibody-drug conjugates against ovarian and pancreatic tumor cell populations  
### Abbvie/StemCentrx in South San Francisco, California
I worked at the start-up, StemCentrx, as a research associate in the cancer biology group. There, I identified tumorigenic subpopulations and validated antibody-drug conjugates targeting those subpopulations for the treatment of ovarian and pancreatic cancer. I led three drug-target campaigns and supported the pre-clinical validation of four targets in ovarian cancer. My work contributed to authorship on four patent applications and four clinical trials for the treatment of ovarian cancer with antibody-drug conjugates. 

**Relevant skills** 
- Tumor modeling: Patient-derived tumor xenograft models, pancreatic and ovarian orthotopic modeling, tumor subpopulation and cancer stem cell characterization, tissue culture, FACS, flow cytometry. 


- Candidate drug-target discovery: Bioinformatic analysis of RNA-seq, mass-spectometry, publically available datasets (i.e. TCGA, COSMIC, CCLE, cBioportal), somatic mutation interpretation, with MySQL, python, excel, and Tableau to identify candidate antibody-drug conjugate targets.


- Pre-clinical antibody-drug conjugate validation: Designed patient-derived xenograft efficacy testing, antibody selection and development, invitro killing assays, development of chemo-resistent xenograft cells.

**Publications**
- [Anti-EFNA4 Calicheamicin Conjugates Effectively Target Triple-Negative Breast and Ovarian Tumor-Initiating Cells to Result in Sustained Tumor Regressions](https://pubmed.ncbi.nlm.nih.gov/26015513/)

## Hematopoietic stem cell responses to inflammatory cytokine signaling
### Passegué lab at UCSF, San Francisco, California
In my undergraduate work, I studied hematopoietic stem cells (HSCs) and cell-fate decisions in response to inflammatory cytokines with Dr. Eric Pietras and Dr. Emmanuelle Passegué at UCSF. My experiments characterized hematopoietic stem cell responses to exposure of inflammatory cytokines, such as IL-1b, IFN-alpha, and IL-10 in vitro. Through a series of experiments, we discovered that in response to pro-inflammatory cytokines, HSCs rapidly differentiate and commit to myeloid cell-fates to replenish downstream myeloid cell populations. This work highlighted the replenishing role of HSCs during inflammation. 

**Relevant skills** 
- Hematopoietic stem cell characterization: isolation from mouse bone marrow, tissue culture, FACS, flow cytometry, proliferation and apoptosis assays, colony formation assays, live-cell image tracking.


- Other: qPCR, mouse husbandry. 

**Publications**
- [Chronic interleukin-1 exposure drives haematopoietic stem cells towards precocious myeloid differentiation at the expense of self-renewal](https://pubmed.ncbi.nlm.nih.gov/27111842/)
- [Re-entry into quiescence protects hematopoietic stem cells from the killing effect of chronic exposure to type I interferons](https://pubmed.ncbi.nlm.nih.gov/24493802/)
